architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -175,29 +175,143 @@ endif::[]
+
+* You can configure security around who can view and use the images through permissions on the image stream objects. 
+
+* Users that lack permission to read or list images on the cluster level can still retrieve the images tagged in a project using image streams.
+
+
+For a curated set of image streams, see the
 link:https://github.com/openshift/library[OpenShift Image Streams and Templates
 library].
 
-For example, if a deployment is using a certain image and a new version of that
-image is created, a deployment could be automatically performed.
+
+// https://access.redhat.com/solutions/3016781 Caveats around referencing latest tags
+When using Docker images it is important to understand what image your image stream tag is pointing to and be aware if the image that the image stream tag is pointing to can change.
+For example, if pointing to the `latest` image stream tag and the version of the `latest` image changes, the new version might not be backwards-compatible with your applications.
+Using an image stream with a version-specific image stream tag will prevent problems using `latest` by providing a stable pointer until the next import.
+
+=== Important terms
+
+Docker repository::
+A central location for uploading and downloading images.
+For example, the OpenShift Jenkins images are in a Docker registry:
++
+----
+docker.io/openshift/jenkins-2-centos7
+----
+
+Docker image::
+A reference to an image for a given external registry. It uses the standard Docker pull specification for its name, for example:
+
+----
+openshift/jenkins-2-centos7
+----
+
+Docker image tag::
+A label applied to a Docker image in a repository that distinguishes a specific image. For example, here *3.6.0* is a tag:
+
+----
+docker.io/openshift/jenkins-2-centos7:3.6.0
+----
+
+Docker image ID::
+A SHA code that can be used to pull an image. For example:
+
+----
+docker.io/openshift/jenkins-2-centos7@sha256:ab312bda324
+----
+
+Image stream::
+An {product-title} object that contains pointers to any number of Docker-formatted container images identified by tags. You can think of an image stream
+as equivalent to a Docker repository.
+
+Image stream tag::
+A named pointer to an image in an image stream. An image stream tag is similar to a Docker image tag. See xref:image-stream-tag[Image Stream Tag] below.
+
+Image stream image::
+An image that allows you to retrieve a specific Docker image from a particular image stream where it is tagged.
+See xref:image-stream-image[Image Stream Images] below.
+
+Image stream trigger::
+A trigger that causes a specific action when an image stream tag changes. For example, importing an image can be a trigger.
+Importing causes a trigger to fire when there are Deployments, Buildss or other resources listening for those. See xref:image-stream-triggers[Image Stream Triggers] below.
```

Comment:
importing is not the trigger.

the trigger is "the value of the tag changed".  importing can cause the value of the tag to change, but so can pushing a new image to the tag.

Reply:
above comment still valid (importing is not a trigger, importing can cause the value of the tag to change, which is a trigger if it happens).  

also s/Buildss/Builds/